pkg/antctl/raw/resourceexport/command.go

```diff
@@ -0,0 +1,127 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//TODO: not done
+
+package clusterset
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	antreamcscheme "antrea.io/antrea/multicluster/pkg/client/clientset/versioned/scheme"
+
+	k8sscheme "k8s.io/client-go/kubernetes/scheme"
+
+	mcsscheme "sigs.k8s.io/mcs-api/pkg/client/clientset/versioned/scheme"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+
+	multiclusterv1alpha1 "antrea.io/antrea/multicluster/apis/multicluster/v1alpha1"
+)
+
+var Command *cobra.Command
+
+func init() {
+	Command = &cobra.Command{
+		Use:     "resourceexport",
+		Aliases: []string{"resourceexports"},
+		Short:   "Print multicluster resourceexports",
+		Args:    cobra.MaximumNArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			var err error
+			if len(args) != 0 {
+				err = runE(cmd, args[0])
+			} else {
+				err = runE(cmd, "")
+			}
+			if err != nil {
+				fmt.Println(fmt.Errorf("please input the correct numbers of argumrnts here"))
+			}
+		},
+	}
+
+}
+
+func runE(cmd *cobra.Command, ClusterID string) error {
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	kubeconfig.GroupVersion = &schema.GroupVersion{Group: "", Version: ""}
+
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+	scheme := runtime.NewScheme()
+	err = mcsscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+	err = antreamcscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+	err = k8sscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: scheme})
+	if err != nil {
+		return err
+	}
+
+	res := &multiclusterv1alpha1.ResourceExportList{}
+
+	err = k8sClient.List(context.TODO(), res, &client.ListOptions{Namespace: metav1.NamespaceAll})
+
+	if err != nil {
+		return err
+	}
+
+	var resExports []multiclusterv1alpha1.ResourceExport
+
+	if ClusterID != "" {
+		for _, r := range res.Items {
+			if r.Labels["sourceClusterID"] == ClusterID {
+				resExports = append(resExports, r)
+			}
+		}
+	} else {
+		resExports = res.Items
+	}
+
+	fmt.Println(output(resExports))
+
+	return nil
+}
+
+func output(resExports []multiclusterv1alpha1.ResourceExport) string {
+	var output strings.Builder
+	formatter := "%-50s%-50s%-50s\n"
+	output.Write([]byte(fmt.Sprintf(formatter, "NAME", "NAMESPACE", "ClusterID")))
+	for _, r := range resExports {
+		fmt.Fprintf(&output, formatter, r.Name, r.Namespace, r.Labels["sourceClusterID"])
```

Comment:
better to place `Namespace` before `Name` like kubectl do. is there is way to guarantee the order of output? I mean it's better to put ClusterID in first column and list all ResourceExport together from one cluster.